### PR TITLE
Add linting and code formatting standardization

### PR DESCRIPTION
**Pull request by HDauven**
**Purpose**
Add linting and code formatting support through ESLint and Prettier to standardize code formatting.

**Issues**
#48

**Changes**
- Add ESLint and Prettier dependencies.
- Add ESLint and Prettier configuration files.

**Check list**
- [x] Code runs without regressions even though new code is incomplete
- [ ] Code reviewer has been explicitly notified outside automatic notification channels

### CODE REVIEW

**Review by MariusTH — APPROVED** [review]

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true,
+        "node": true
+    },
+    "extends": ["eslint:recommended", "prettier"],
```
Comment (jijordre) [inline .eslintrc:7]: Did you consider extending `@nahmii` (from github:nahmii/eslint-config) rather than `eslint:recommended`?

**Review by jijordre — APPROVED** [review]
LGTM, but still needed to raise the question about which ESLint config to extend. 😄

File: .eslintrc (hunk 1 above)
Comment (HDauven) [reply .eslintrc]: Ran into some issues here. Had to explicitly import `minimatch` or some weird reason. 

Latest commits make it work though. It uses the Nahmii style guide and prettier to format everything properly.

**Review by MariusTH — APPROVED** [review]
LGTM!

**Review by jijordre — APPROVED** [review]

**Pull request merged**